packages/patrol/lib/src/common.dart

```diff
@@ -55,8 +55,9 @@ void patrolTest(
   finders.PatrolTesterConfig config = const finders.PatrolTesterConfig(),
   NativeAutomatorConfig nativeAutomatorConfig = const NativeAutomatorConfig(),
   @Deprecated('''
-This variable will be removed in the future, 
-if you use nativeAutomation with false, we recommend using patrolWidgetTest()''')
+This variable will be removed in the future. 
+If you use nativeAutomation with false, we recommend using patrolWidgetTest().
+See https://patrol.leancode.co/patrol-finders-release for more information.''')
```

Comment:
People that have this in the default `false` state will never see the deprecation message.